datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -262,13 +262,42 @@ func (f *ChainDataFetcher) stopRangeFetching() error {
 	return nil
 }
 
+func (f *ChainDataFetcher) makeChainEventWithRetry(blockNumber uint64) (blockchain.ChainEvent, error) {
+	var (
+		retryMax      = 100
+		retryInterval = 1 * time.Second
+	)
+	ev, err := f.makeChainEvent(blockNumber)
+	for retryCount := 1; err != nil; retryCount++ {
+		select {
+		case <-f.stopCh:
+			return ev, err
+		default:
+			if retryCount < retryMax {
+				logger.Warn("retrying to make a chain event...", "blockNumber", blockNumber, "retryCount", retryCount, "retryMax", retryMax, "err", err)
+			}
+
+			// leaves an error log periodically after max retry
+			if retryCount%retryMax == 0 {
+				logger.Error("max retry exceeded. retrying to make a chain event...", "blockNumber", blockNumber, "retryCount", retryCount, "retryMax", retryMax, "err", err)
```

Comment:
```suggestion
				logger.Warn("max retry exceeded. retrying to make a chain event...", "blockNumber", blockNumber, "retryCount", retryCount, "retryMax", retryMax, "err", err)
			}

			// leaves an error log periodically after max retry
			if retryCount%retryMax == 1 {
				logger.Error("retrying to make a chain event...", "blockNumber", blockNumber, "retryCount", retryCount, "retryMax", retryMax, "err", err)
```
* I think the log is switched
* Changing 0 to 1 could make the log at the first try.

Reply:
@winnie-byun Umm. I think the first retries warn that it is retrying. After that, the max retries exceeded and we consider it error situation, so it leaves error log.

Reply:
@jeongkyun-oh Sorry My bad